### PR TITLE
STY:  formatted the code to be more readable in numpy.core

### PR DESCRIPTION
**Pull request by hammad-ahmed-1999**
did not add or remove anything, added white spaces to align the code

### CODE REVIEW

File: numpy/core/src/umath/umathmodule.c [hunk 1]
```diff
@@ -216,45 +217,45 @@ add_newdoc_ufunc(PyObject *NPY_UNUSED(dummy), PyObject *args)
...
-NPY_VISIBILITY_HIDDEN PyObject *npy_um_str_casting = NULL;
-NPY_VISIBILITY_HIDDEN PyObject *npy_um_str_order = NULL;
-NPY_VISIBILITY_HIDDEN PyObject *npy_um_str_dtype = NULL;
-NPY_VISIBILITY_HIDDEN PyObject *npy_um_str_subok = NULL;
-NPY_VISIBILITY_HIDDEN PyObject *npy_um_str_signature = NULL;
-NPY_VISIBILITY_HIDDEN PyObject *npy_um_str_sig = NULL;
-NPY_VISIBILITY_HIDDEN PyObject *npy_um_str_extobj = NULL;
-NPY_VISIBILITY_HIDDEN PyObject *npy_um_str_array_prepare = NULL;
-NPY_VISIBILITY_HIDDEN PyObject *npy_um_str_array_wrap = NULL;
+NPY_VISIBILITY_HIDDEN PyObject *npy_um_str_out            = NULL;
```
Comment (charris) [inline numpy/core/src/umath/umathmodule.c:220]: No, we don't do alignment.

File: numpy/core/src/umath/umathmodule.c [hunk 2]
```diff
@@ -35,11 +35,11 @@ static PyUFuncGenericFunction pyfunc_functions[] = {PyUFunc_On_Om};
...
-                                NPY_CASTING casting,
-                                PyArrayObject **operands,
-                                PyObject *type_tup,
-                                PyArray_Descr **out_dtypes)
-{
+                            NPY_CASTING casting,
+                            PyArrayObject **operands,
+                            PyObject *type_tup,
+                            PyArray_Descr **out_dtypes
+){
```
Comment (charris) [inline numpy/core/src/umath/umathmodule.c:42]: Pleas keep `)` on the preceding line.

**Comment by charris** [comment]
You should read `doc/C_STYLE_GUIDE.rst.txt` before attempting this sort of thing. We also prefer such style fixes to be part of PRs addressing other problems these days.

**Comment by seberg** [comment]
Thanks for the interest in contributing. I am going to close this, since only the first function argument indentation may be desirable as per our style-guide.

**Pull request closed without merging**